core-services/image-mirroring/openshift/mapping_origin_4_5

```diff
@@ -163,6 +163,7 @@ registry.svc.ci.openshift.org/origin/4.5:multus-cni quay.io/openshift/origin-mul
 registry.svc.ci.openshift.org/origin/4.5:multus-route-override-cni quay.io/openshift/origin-multus-route-override-cni:4.5 quay.io/openshift/origin-multus-route-override-cni:4.5.0 quay.io/openshift/origin-multus-route-override-cni:latest
 registry.svc.ci.openshift.org/origin/4.5:multus-whereabouts-ipam-cni quay.io/openshift/origin-multus-whereabouts-ipam-cni:4.5 quay.io/openshift/origin-multus-whereabouts-ipam-cni:4.5.0 quay.io/openshift/origin-multus-whereabouts-ipam-cni:latest
 registry.svc.ci.openshift.org/origin/4.5:must-gather quay.io/openshift/origin-must-gather:4.5 quay.io/openshift/origin-must-gather:4.5.0 quay.io/openshift/origin-must-gather:latest
+registry.svc.ci.openshift.org/origin/4.5:network-metrics-daemon quay.io/openshift/origin-network-metrics-daemon:4.5 quay.io/openshift/origin-network-metrics-daemon:4.5.0 quay.io/openshift/origin-network-metrics-daemon:latest
```

Comment:
Will *every* cluster have these metrics?  I.e. no matter what these metrics will always be returned in every 4.5 cluster?  Or are the metrics within this deployed only in specific scenarios?

Reply:
@smarterclayton  as things are today, every cluster would have these metrics. 
The current idea is to have CNO ship this under the same parameter that deploys multus (which is enabled by default).

Reply:
@smarterclayton does this make sense? Can we push this forward?